utils/miller/wrapper.py

```
@@ -0,0 +1,81 @@
+        io_opts += f" --prepipe {prog}"
+        break
+
+# Delimiter
+for in_format in io_formats:
+    if get_format(inputs) == in_format:
+        io_opts += f" --i{in_format}"
+        break
+
+
+### OUTPUT
+output = f"> {snakemake.output[0]}"
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)